en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,183 @@
+ 
+- Added new @aggregation-status attribute for the indicator element (`#363 <https://github.com/IATI/IATI-Schemas/issues/363>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new location element (as a child of the baseline element) (`#365 <https://github.com/IATI/IATI-Schemas/issues/365>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new dimension element (as a child of the baseline element) (`#366 <https://github.com/IATI/IATI-Schemas/issues/366>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
+  
+Organisations Schema
+-------------------
+
+- Updated definition: organisation-identifier element (`#361 <https://github.com/IATI/IATI-Schemas/issues/361>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+
+Activities and Organisations Schemas
+-------------------
+  
+- Updated definition: all @xml:lang attributes (`#336 <https://github.com/IATI/IATI-Schemas/issues/336>`__) (`discussion link <https://discuss.iatistandard.org/t/language-recommend-use-of-iso-639-1-included-2-03/842>`__)
+
+- Updated definition: [reporting/participating/receiver/provider]-org @ref attribute (`#339 <https://github.com/IATI/IATI-Schemas/issues/339>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+
+- Updated definition: reporting-org @secondary-reporter attribute (`#340 <https://github.com/IATI/IATI-Schemas/issues/340>`__) (`discussion link <https://discuss.iatistandard.org/t/modify-definition-of-secondary-publisher-included-2-03/846>`__)
+ 
+- Updated definition: [recipient-region/recipient-region-budget/sector AND transaction/recipient-region] @vocabulary-uri attribute (`#341 <https://github.com/IATI/IATI-Schemas/issues/341>`__) (`discussion link <https://discuss.iatistandard.org/t/guidance-on-u-r-i-usage-for-publisher-s-own-vocabularies-included-2-03/850>`__)
+
+- Added new description element (as a child of all document-link elements) (`#357 <https://github.com/IATI/IATI-Schemas/issues/357>`__) (`discussion link <https://discuss.iatistandard.org/t/document-link-description-included-2-03/841>`__)
+
+.. _2_03_codelist_changes:
+
+Codelist Changes
+================
+
+Embedded to Non-Embedded:
+------------------------
+
+The following codelists have moved from Embedded to Non-Embedded (`#114 <https://github.com/IATI/IATI-Codelists/issues/114>`__) (`Discussion link <https://discuss.iatistandard.org/t/redefine-selected-codelists-as-non-embedded-included-2-03/854>`__):
```

Comment:
Add something like `Alongside 2.03` (it's a change, but not part of 2.03 itself since it's also impacting previous versions).

Reply:
222bbe026dffa2f4ed8c330218d689895667683c